index.rst

```diff
@@ -5,77 +5,331 @@
-This service has been referred  to as Catcher.
-In essence, the Catcher is not that different from the Watcher or other stream processing off-the-shelf technologies like Kapacitor.
-This technote explores the idea of using InfluxDB OSS 2.x new task engine as an alternative implementation for the Catcher.
+This technote contains design considerations for the Catcher, an event-driven processing component for the Rubin Observatory Control System.
+We briefly review the existing functionality of the system and discuss the use case for the Catcher.
+The idea originated from the First-Look Analysis and Feedback Functionality (FAFF) working group, and a more in-depth analysis of the use cases can be found in their reports.
+
 
 Introduction
 ============
 
 One of the finding of the First-Look Analysis and Feedback Functionality (FAFF) working group :sitcomtn:`025` :cite:`SITCOMTN-025`, was the need for a service to ...
 
-   handle the low-level coordiation of identifying when a specific condition is met, then launching and monitoring an analysis process.
+   handle the low-level coordination of identifying when a specific condition is met, then launching and monitoring an analysis process.
 
-The original proposal from the FAFF working group was to develop a Commandable SAL Component (CSC), similar to the Watcher, that would provide be capable of monitoring data from the system and trigger certain actions (e.g. perform some computation and/or generate reports) based on some pre-defined conditions.
+The idea is basically to have a component capable of executing event-driven processing and report generation.
 
-Following additional discussions, we have decided to experiment with an alternative approach, replacing the CSC with an off-the-shelf solution provided by provided by Sasquatch (see :sqr:`068` :cite:`SQR-068`).
+The Rubin Observatory Control System currently contains a ``Watcher`` CSC that is capable of generating alerts from user defined rules (e.g. event-driven alerts).
+Moreover, the ``ScriptQueue`` and ``OCPS`` CSCs are both able to execute command-driven processes.
+While the ``ScriptQueue`` is tailored towards control system operations (e.g. moving the telescope, taking data, etc.), the ``OCPS`` is dedicated to executing DM-pipeline tasks processes.
+The missing piece of functionality is the capability of mixing a ``Watcher``-like behavior (event-processing) with ``ScriptQueue``/``OCPS``-like behavior.
+Furthermore, we are also interested in generating comprehensive reports from the results of these processes and alarms for particular conditions.
 
-As discussed in :sqr:`068` :cite:`SQR-068`, Sasquatch is based on InfluxDB OSS 2.x, a time series database. The new version of InfluxDB brings a number of important updates, amongst which two are relevant for stream processing; `Flux`_ and `InfluxDB task`_.
+The following are the key functionality we are looking for in the ``Catcher``.
```

Comment:
uhh... I guess I was expecting a list when written like this? 

Did you leave this here by accident or is this meant to be an intro to the following sections?

Reply:
reworded...